docs/Gallery.md

```diff
@@ -60,3 +60,21 @@ arrows depict the shift vector. Image credit: Alex Carpenter, CSUF
 </figcaption>
 </figure>
 \endhtmlonly
+
+## Binary black hole initial data
+
+SpECTRE can generate initial data to start simulations of merging black holes.
+This problem involves solving the elliptic constraint sector of the Einstein
+equations for a slice of spacetime that contains two black holes with the
+requested parameters. SpECTRE uses the \ref ::Xcts "XCTS" formulation with a
+non-conformally-flat background defined by the
+\ref ::Xcts::AnalyticData::Binary "superposed Kerr-Schild" formalism
```

Comment:
Should this link to Kerr-Schild docs?

Reply:
No this is the documentation for the "superposed" part of the story

Reply:
How about hyperlinking the word "superposed" to the Binary, and then a separate hyperlink for "Kerr-Schild" to the Kerr-Schild docs? If I was new to reading this, I would have assumed clicking on "superposed Kerr-Schild" would include something about Kerr-Schild, but the Binary docs don't.

Reply:
If you're ok with the current state then I'd just keep it like this